clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -77,7 +78,7 @@ public class CommonClientConfigs {
 
     public static final String CLIENT_RACK_CONFIG = "client.rack";
     public static final String CLIENT_RACK_DOC = "A rack identifier for this client. This can be any string value which indicates where this client is physically located. It corresponds with the broker config 'broker.rack'";
-    public static final String DEFAULT_CLIENT_RACK = "";
+    public static final String CLIENT_RACK_DEFAULT = "";
```

Comment:
Those configs are exposed publicly, so maybe we need a KIP to deprecate them?

Reply:
This is a good point. I missed that the default values also part of this.  I'll revert these for now and address it later with a KIP.

Reply:
do we need a KIP to move the default values to `CommonClientConfigs` as well?

Reply:
> do we need a KIP to move the default values to CommonClientConfigs as well?

yep ! see the following KIP rules:
```
What is considered a "major change" that needs a KIP?
...
Any change that impacts the public interfaces of the project
All of the following are public interfaces that people build around:
...
Configuration, especially client configuration
...
New or Changed Public Interfaces: impact to any of the "compatibility commitments" described above. We want to call these out in particular so everyone thinks about them.
```

Reply:
had a second thought about this and instead of raising a KIP I decided to just move all metric configs in one place for now in `MetricConfigs`. As I want to keep KAFKA-15853 as simple as possible as it already huge effort to move these config out of core. I will create a cleanup Jiras to align the naming convention for some of these config files.

Reply:
Created this jira https://issues.apache.org/jira/browse/KAFKA-16638 as a followup to align the naming convention for config classes. Left note there that some classes need KIPs and might need a separate jira later.